src/main/kotlin/model/MoJSignOn.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+
+class MoJSignOn private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var app: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem("MoJ SignOn", "Allows users to login into digital services")
+
+      app = system.addContainer("MoJ SignOn Webapp", "OAuth based single sign on component designed to authorise staff access to multiple applications with a single account", "Ruby on Rails").apply {
+        setUrl("https://github.com/ministryofjustice/moj-signon")
+        Tags.WEB_BROWSER.addTo(this)
+        Heroku.dyno.add(this)
+      }
+
+      val db = system.addContainer("Database", "Contains authentication and authorisation records", "PostgreSQL").apply {
+        Tags.DATABASE.addTo(this)
+        Heroku.database.add(this)
+      }
+
+      app.uses(db, "connects to")
+    }
+
+    override fun defineRelationships() {
+    }
+
+    override fun defineViews(views: ViewSet) {
+    }
+  }
+}
```

Comment:
**Short version** 100%; I'd create follow-up PRs that introduce 'topical' landscape diagrams and maybe lifecycle status or intent.

**Long version**
In the last development session, one of the feedback was about making it easier to discover things.

I was thinking that maybe removing the system context diagrams and introducing topical landscape diagrams would help with that discovery process.

For example, instead of having MoJ SignOn context, HMPPS Auth context, we could have an "Authentication landscape" which could detail these lifecycle intentions.

Reply:
Sorry, to answer the question: I don't know, I think it is generally accepted that we need to switch to HMPPS Auth, but I haven't seen/searched for a decision about it yet.